notebook.ipynb

```diff
@@ -1366,13 +1541,68 @@
+      "text/plain": [
+       "<Figure size 1000x1000 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
     "# AnnualIncome by AcademicLevel\n",
+    "analysisData = pd.DataFrame({\n",
+    "    'AcademicLevel': data['AcademicLevel'],\n",
+    "    'AnnualIncome': data['AnnualIncome'],\n",
+    "})\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "sns.boxplot(x='AcademicLevel', y='AnnualIncome', data=analysisData)\n",
+    "plt.title('Distribution of AnnualIncome across AcademicLevels')\n",
+    "plt.xlabel('Academic Level')\n",
+    "plt.ylabel('Annual Income ($)')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 191,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Mann-Whitney U Test:\n",
+      "Statistic: nan\n",
+      "p-value: nan\n",
```

Comment:
Kindly investigate the reason for `p-value` not being printed.

Also, Print 
```bash
Null Hypothesis:
Alternate Hypothesis:
```
before the actual test for a better readability of outcome.